src/handlers.ts

```diff
@@ -62,7 +69,7 @@ export class MyHandler extends Handler {
       }
       if (info_Icons.length === 0) {
```

Comment:
info_Icons appears to be set to [] and never updated during this function so this will always be true. Also infoIcon_initial is never reassigned, so why change the const?

Reply:
I have reverted back this change.